test/runner_test.lua

```diff
@@ -55,6 +55,10 @@ g.test_repeat = function()
     t.assert_equals(run_file('flaky.lua', '-r 2'), 256)
 end
 
+g.test_group_repeat = function()
+    t.assert_equals(run_file('flaky_group.lua', '--repeat 2'), 256)
+end
+
```

Comment:
Can we add a test verifying that when providing `-gr 2 -r 2` we get expected behavior?